src/main/java/com/czertainly/core/service/scep/impl/ScepServiceImpl.java

```diff
@@ -439,7 +438,7 @@ private ScepResponse issueCertificate(ScepRequest scepRequest, IntuneScepService
         }
         ClientCertificateSignRequestDto requestDto = new ClientCertificateSignRequestDto();
         try {
-            requestDto.setPkcs10(new String(Base64.getEncoder().encode(scepRequest.getPkcs10Request().getEncoded())));
+            requestDto.setRequest(new String(Base64.getEncoder().encode(scepRequest.getPkcs10Request().getEncoded())));
```

Comment:
I do not think the default `format` will be assigned here because we are not using builder as with other DTOs. It should be set to `pkcs10` in case of SCEP, SCEP does not work with `crmf`.